src/pyodide/helpers.bzl

```diff
@@ -131,12 +131,9 @@ def pyodide_static():
         [
             "internal/*.ts",
             "internal/topLevelEntropy/*.ts",
-            # The pool directory is only needed by typescript, it shouldn't be used at runtime.
```

Comment:
Changes in this file were added to work around Windows compile errors. I did not check if it is safe to drop them after adding the workaround, but I think they are a win either way, basically dropping build inputs where we don't need them.

Reply:
@hoodmane